modules/secrets-store-google.adoc

```diff
@@ -0,0 +1,326 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-secrets-store.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="secrets-store-google_{context}"]
+= Mounting secrets from Google Secret Manager
+
+You can use the {secrets-store-operator} to mount secrets from Google Secret Manager to a CSI volume in {product-title}. To mount secrets from Google Secret Manager, your cluster must be installed on {gcp-first}.
+
+// TODO: Confirm must be on GCP
```

Comment:
@ropatil010 Is this a true statement above that their cluster has to be installed on GCP?

Reply:
Hi @bergerhoffer TBH the cases work on other platform like(AWS) as well. As we followed for previous things like Azure keyvault with Azure platform, AWS secrets manager with AWS so i suggested in the same way like GCP secrets manager with GCP platform. Need to get inputs from Anjali on this.

Reply:
Thanks I'll check with Anjali.